EIPS/eip-epbs.md

```diff
@@ -0,0 +1,260 @@
+
+## Motivation
+
+This EIP solves a different set of unrelated important problems. 
+
+- An overwhelming majority of beacon block proposers outsource the construction of the execution payload within their blocks to a third party (henceforth called a *builder*). In order to do so, they request the hash tree root (HTR) of a promised execution payload and submit a `SignedBlindedBeaconBlock` to a trusted party that is tasked with replacing the HTR with the full execution payload (received from the builder) before broadcasting. This EIP allows for a trust-free direct communication between the beacon block proposer and the builder, guaranteeing that an honest beacon block proposer will receive payment from the builder regardless of the latter actions and that the honest builder's payload will be the canonical head of the chain regardless of the proposer's action. 
+- Currently, validators have the time between receiving the full beacon block (including an execution payload) and the attesting deadline (4 seconds in Ethereum mainnet) to perform both consensus and execution state transition functions, check blob data availability and evaluate the new head of the blockchain. The remainder of the slot time is spent during less CPU-intense and critical task. By separating the validation of the execution and consensus part of the block, validators are only tasked to perform the consensus state transition function in this critical time before attesting, while execution and data availability validation is deferred for most of the remainder of the slot (between the builder's reveal time and the next attestation deadline). 
+- By removing the full execution payload size from the consensus block, it allows for faster network propagation on the critical path. 
+- It removes the increased reorg likeliness of including blob transactions in blocks. 
```

Comment:
Feel like one level of details lower would be good here. Why is this claim true?

Reply:
Builder can send the blobs before seeing attestations. Thanks will add this